tests/foreman/cli/test_reporttemplates.py

```diff
@@ -857,3 +857,37 @@ def test_positive_generate_entitlements_report_multiple_formats(self):
             })
             assert vm.hostname in result_csv[1]
             assert self.setup_subs_id[0]['name'] in result_csv[1]
+
+    @tier3
+    def test_positive_schedule_Entitlements_report(self):
+        """Schedule an report using the Entitlements template in csv format.
+
+        :id: 572fb387-86e0-40e2-b2df-e8ec26433610
+
+
+        :setup: Installed Satellite with Organization, Activation key,
+                Content View, Content Host, and Subscriptions.
+
+        :steps:
+            1. hammer report-template schedule --organization '' --id '' --report-format ''
+
+        :expectedresults: report is scheduled and generated containing all the expected information
+                          regarding Entitlements.
+
+        :CaseImportance: High
+        """
+        with VirtualMachine(distro=DISTRO_RHEL7) as vm:
+            vm.install_katello_ca()
+            vm.register_contenthost(self.setup_org['label'], self.setup_new_ak['name'])
+            assert vm.subscribed
+            scheduled_csv = ReportTemplate.schedule({
+                'id': 115,
+                'organization': self.setup_org['name'],
+                'report-format': 'csv'
+            })
+            data_csv = ReportTemplate.report_data({
+                'id': 115,
+                'job-id': scheduled_csv[0].split("Job ID: ", 1)[1]
+            })
+            assert vm.hostname in data_csv[1]
+            assert self.setup_subs_id[0]['name'] in data_csv[1]
```

Comment:
Usual comment about hardcoded numbers. Would it make sense to rewrite that to loop to make it more robust? We don't want test to fail just because order of items is slightly different.